CHANGELOG.md

```diff
@@ -12,5 +12,5 @@ First release! What's possible with this first release:
 - Store data back to Solid Pods.
 - Read data from datasets.
 - Manipulate data in datasets.
-- Inspect a user's permissions w.r.t. a given Resource. (Experimental.)
-- Retrieve, delete and/or write any file (including non-RDF) from/to a Pod.
+- Inspect a user's and public permissions w.r.t. a given Resource. (Experimental.)
```

Comment:
Remove the word 'and'...?

Reply:
That would be incorrect: you can inspect public permissions and user-specific permissions separately. There's no such thing as "public user permissions". Not sure if there's a clearer way to word that?